api-reference/10 UI Components/dxScheduler/1 Configuration/resources/label.md

```diff
@@ -5,9 +5,17 @@ default: ''
 ---
 ---
 ##### shortDescription
-Specifies the label of the Appointment popup window field that allows end users to assign a resource of this kind.
+Specifies a resource label.
 
 ---
+Scheduler displays resource labels in the appointment edit form.
+
+To customize the appointment edit form and add resource items in the **editing**.**form**.[items[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/#items) array, implement resource labels in item names. Scheduler generates predefined editing form items as follows:
```

Comment:
I'm not sure what you mean by "implement resource labels in item names". The word "implement" is what confuses me most, I guess. I would understand if you said "specify resource labels as item names" or "use resource labels in item names". 

So you can:

- Add an item to `editing.form.items[]`.
- Set `item.name = resource.label`.

And this way you added a resource item to the edit form. 

```suggestion
To customize the appointment edit form and add resource items in the **editing**.**form**.[items[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/#items) array, specify resource labels as item names. Scheduler generates predefined editing form items as follows:
```